src/lib/browse/layers/lines/PavementWidths.svelte

```diff
@@ -82,27 +108,33 @@
       visibility: show ? "visible" : "none",
     }}
   >
-    <Popup let:props>
+    <Popup let:props openOn="hover">
       {#if props.left_average}
```

Comment:
My overzealous DRY instincts are kicking in, is it overkill to turn this into its own component rather than having mostly the same code for both left and right?